conf/messages

```diff
@@ -101,6 +101,127 @@ upload.progress.error.general=Something has gone wrong, click <a href="/series">
+accessibility.header2=How accessible this website is
+accessibility.body6=We know some parts of this website are not fully accessible:
+accessibility.item6=[items that aren’t accessible]
+accessibility.header3=Feedback and contact information
+accessibility.body7=If you need information on this website in a different format, for example accessible PDF, large print, easy read or audio recording, email [TDR HELP EMAIL]
+accessibility.body8=You can also use these contact details to report accessibility problems with this website.
+accessibility.body9=If you have an accessibility need and would like to participate in user testing as we make improvements to our website, email uxresearch@nationalarchives.gov.uk.
+accessibility.header4=Feedback, contact information and reporting accessibility problems
+accessibility.body10=We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: [provide both details of how to report these issues to your organisation, and contact details for the unit or person responsible for dealing with these reports].
+accessibility.header5=Enforcement procedure
+accessibility.body11=The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you are not happy with how we respond to your complaint, contact the Equality Advisory and Support Service (EASS).
+accessibility.header6=Contacting us or visiting in person
+accessibility.body12=You can contact us by email. We provide information about visiting in person. People with health conditions or impairments that may impact their visit can find out more.
+accessibility.header7=Technical information about this website’s accessibility
+accessibility.body13=The National Archives is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+accessibility.header8=Compliance status
+accessibility.body14=This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances and exemptions listed below:
+accessibility.subheader1=Non-accessible content
+accessibility.body15=Based on sample testing of pages selected by our user experience and metrics teams, we know that the content listed below is non-compliant with the accessibility regulations for the following reasons:
+accessibility.subheader2=Non-compliance with the accessibility regulations
+accessibility.item7=[list issues here]
```

Comment:
I'm going to see if I can catch Kirsten for a quick chat about these pages. I want to know how she feels about deploying them while they're still in progress, or if it would be better to wait until we've written the full statement and asked Ashley to review.

Reply:
Yess, it's a bit tricky because even though the pages exist with the drafts in both, it's an ongoing process at the moment. Ashley is working on finishing the accessibility statement, so that might be something which can be deployed (help guide is still a WIP as we don't have the guidance video etc yet)

Reply:
Just unresolving this comment because when I talked to Kirsten last week, she said that she only wants to deploy these changes once we've cleared up all of the to-do comments. Did she talk to you about it as well?

Reply:
(removed the discussed items from this PR)